app/helpers/data_tables_helper.rb

```diff
@@ -49,7 +49,7 @@ def browse_tabs
       plant_populations: data_tables_path(model: :plant_populations),
       trait_descriptors: data_tables_path(model: :trait_descriptors, group: true),
       linkage_maps: data_tables_path(model: :linkage_maps),
-      qtl: data_tables_path(model: :qtl)
+      qtl: data_tables_path(model: :qtl, group: true)
```

Comment:
It's explained in index.html.haml. With this switch on, no per-record actions are rendered in the table. Since single table row corresponds to many records, no per-record actions make sense.